fabcar/javascript/fileUploadServer.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express')
+const app = express()
+const bodyParser = require('body-parser')
+const cors = require('cors')
+const port = 3000
+const HOST = 'localhost'
+'use strict';
+
+const { Gateway, Wallets } = require('fabric-network');
+const path = require('path');
+const fs = require('fs');
+const FabricCAServices = require('fabric-ca-client');
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(cors());
+
+var fileUpload = require('/home/shatabdi/Health-Blockchain/fabcar/javascript/uploadToDrive/encryptAndUpload.js')
```

Comment:
replace with relative path, not absolute path. Otherwise it will only work on your PC.
```suggestion
var fileUpload = require('uploadToDrive/encryptAndUpload.js')
```